src/components/users/UserList.js

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from "react"
+import { useNavigate } from 'react-router-dom'
+import { getUsers, getUsersBySearchTerm } from "../../managers/users"
+import { User } from "./User"
+import "./Users.css"
+// import { UserSearch } from "./UserSearch"
```

Comment:
No commented out code should go into the main branch

Reply:
removed.